rustls/tests/server_cert_verifier.rs

```diff
@@ -8,11 +8,25 @@ mod common;
 use std::sync::Arc;
 
 use common::{
-    do_handshake, do_handshake_until_both_error, make_client_config_with_versions,
-    make_pair_for_arc_configs, make_server_config, ErrorFromPeer, MockServerVerifier,
-    ALL_KEY_TYPES,
+    client_config_builder, client_config_builder_with_versions, do_handshake,
+    do_handshake_until_both_error, do_handshake_until_error, make_client_config_with_versions,
+    make_pair_for_arc_configs, make_server_config, server_config_builder, transfer_altered,
+    Altered, ErrorFromPeer, KeyType, MockServerVerifier, ALL_KEY_TYPES,
 };
-use rustls::{AlertDescription, Error, InvalidMessage};
+use pki_types::{CertificateDer, ServerName};
+use rustls::client::danger::{HandshakeSignatureValid, ServerCertVerified};
+use rustls::client::{danger::ServerCertVerifier, WebPkiServerVerifier};
```

Comment:
sorry, last minute nit: I think the `danger::ServerCertVerifier` import could get pulled up into L17 alongside `HandshakeSignatureValid` and `ServerCertVerified`.

Reply:
On it!